pdf/index.js

```diff
@@ -0,0 +1,71 @@
+// can import this from json
+const data = [
+  {
+    name: 'Naval',
+    twitterHandle: 'naval',
+    image:
+      'https://pbs.twimg.com/profile_images/1256841238298292232/ycqwaMI2_400x400.jpg',
+    createAt: '2018-05-31',
+    tweet: 'How to Get Rich (without getting lucky):'
+  }
+];
+
+// setting available fonts
+var fonts = {
+  Courier: {
+    normal: 'Courier',
+    bold: 'Courier-Bold',
+    italics: 'Courier-Oblique',
+    bolditalics: 'Courier-BoldOblique'
+  },
+  Helvetica: {
+    normal: 'Helvetica',
+    bold: 'Helvetica-Bold',
+    italics: 'Helvetica-Oblique',
+    bolditalics: 'Helvetica-BoldOblique'
+  },
+  Times: {
+    normal: 'Times-Roman',
+    bold: 'Times-Bold',
+    italics: 'Times-Italic',
+    bolditalics: 'Times-BoldItalic'
+  },
+  Symbol: {
+    normal: 'Symbol'
+  },
+  ZapfDingbats: {
+    normal: 'ZapfDingbats'
+  }
+};
+
+var PdfPrinter = require('pdfmake');
+var printer = new PdfPrinter(fonts);
+var fs = require('fs');
+
+// document
+var docDefinition = {
```

Comment:
Could be `const`, please look through other instances too :)

Reply:
done